styles.css

```diff
@@ -0,0 +1,109 @@
+*,
+*::after,
+*::before {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-size: 1.6rem;
+}
+
+.roboto { font-family: 'Roboto', sans-serif; }
+.crete { font-family: 'Crete Round', serif; }
+.poppins { font-family: 'Poppins', sans-serif; }
+.inter { font-family: 'Inter', sans-serif; }
+
+.white { color: #fff; }
+.nBlue1 { color: #091e42; }
+.grey { color: #f4f5f7; }
+.orange { color: #ff6b00; }
+.blackish { color: #1c1a19; }
+.nBlue2 { color: #172b4d; }
+.white1 { color: #fff5e1; }
+
+.bcOrange { background-color: #ff6b00; }
+.bcGrey { background-color: #3c3a39; }
+.bcGrey1 { background-color: #3e3c3c; }
+.bcBlackish { background-color: #1c1a19; }
+
+.w400 { font-weight: 400; }
+.w500 { font-weight: 500; }
+.w700 { font-weight: 700; }
+
+.s15 { font-size: 0.938rem; }
+.s16 { font-size: 1rem; }
+.s17 { font-size: 1.063rem; }
+.s18 { font-size: 1.125rem; }
+.s22 { font-size: 1.375rem; }
+.s24 { font-size: 1.5rem; }
+.s32 { font-size: 2rem; }
+.s40 { font-size: 2.5rem; }
+.s56 { font-size: 3.5rem; }
+.s72 { font-size: 4.5rem; }
+
+.lh20 { line-height: 1.25rem; }
+.lh24 { line-height: 1.5rem; }
+.lh32 { line-height: 2rem; }
+.lh43 { line-height: 2.688rem; }
+.lh44 { line-height: 2.75rem; }
+.lh52 { line-height: 3.25rem; }
+.lh64 { line-height: 4rem; }
+.lh88 { line-height: 5.5rem; }
+
+.ls048 { letter-spacing: 0.03rem; }
+.ls037 { letter-spacing: 0.023rem; }
+
+#main {
+  display: flex;
+  flex-direction: column;
+}
+
+.statusBar {
+  display: flex;
+  justify-content: center;
+  height: 44px;
+  align-items: center;
+}
+
+.statusBarContent {
+  display: flex;
+  width: 316px;
+  justify-content: space-between;
+}
+
+.navigation {
+  display: flex;
+  height: 48px;
+  justify-content: center;
+  align-items: center;
+}
+
+.navigationContent {
+  display: flex;
+  width: 316px;
+  justify-content: space-between;
+}
+
+.headline {
+  display: flex;
+  height: 590px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: #1c1a19 url("img/introBkg.svg") no-repeat;
+  background-position: 100% 100%;
+}
```

Comment:
- [ ] Kindly adjust the height of your design to look as close as possible to your chosen Figma design 🙏 
- [ ] Please make your text bigger so it looks as close as possible to your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/205104290-0a45597c-2113-4fa6-8666-0e6793001e09.png)  |  ![image](https://user-images.githubusercontent.com/74814780/205104249-e26e9361-6e9a-48c4-a457-5edf8a64edf2.png)

Reply:
Fixed.